py/src/ai/h2o/sparkling/ml/params/H2OTargetEncoderParams.py

```diff
@@ -61,24 +60,24 @@ class H2OTargetEncoderParams(Params):
     blendedAvgEnabled = Param(
         Params._dummy(),
         "blendedAvgEnabled",
-        "If set, the target average becomes a weighted average of the posterior average for a given "
-        "categorical level and the prior average of the target. The weight is determined by the size "
-        "of the given group that the row belongs to. By default, the blended average is disabled.",
+        """If set, the target average becomes a weighted average of the posterior average for a given
+           categorical level and the prior average of the target. The weight is determined by the size
```

Comment:
Wouldn't the indentation be included within the text?

Reply:
yes, it probably will, I was following same indentation pattern as above ( keep align to first sentence)